extract_assets.py

```diff
@@ -143,14 +150,21 @@ def main():
                 if file.endswith(".xml"):
                     xmlFiles.append(fullPath)
 
+        class CannotMultiprocessError(Exception):
+            pass
+
         try:
             numCores = int(args.jobs or 0)
             if numCores <= 0:
                 numCores = 1
             print("Extracting assets with " + str(numCores) + " CPU core" + ("s" if numCores > 1 else "") + ".")
-            with multiprocessing.get_context("fork").Pool(numCores,  initializer=initializeWorker, initargs=(mainAbort, args.unaccounted, extractedAssetsTracker, manager)) as p:
+            try:
+                mp_context = multiprocessing.get_context("fork")
+            except ValueError as e:
+                raise CannotMultiprocessError() from e
+            with mp_context.Pool(numCores,  initializer=initializeWorker, initargs=(mainAbort, args.unaccounted, extractedAssetsTracker, manager)) as p:
                 p.map(ExtractFunc, xmlFiles)
-        except (multiprocessing.ProcessError, TypeError):
+        except (multiprocessing.ProcessError, TypeError, CannotMultiprocessError):
```

Comment:
windows thing idk, this fixes:
![image](https://user-images.githubusercontent.com/5306619/235380776-acd389f8-ac96-4452-a16d-bf6fd3ec12c7.png)